build.gradle

```diff
@@ -1,86 +1,81 @@
 plugins {
-	id 'fabric-loom' version '0.5-SNAPSHOT'
-	id 'maven-publish'
+    id 'fabric-loom' version '0.8-SNAPSHOT'
+    id 'maven-publish'
 }
-
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
+repositories {
+    maven { url 'https://maven.fabricmc.net/' }
+    maven { url 'https://maven.shedaniel.me/' }
+    maven { url 'https://maven.terraformersmc.com/releases/'}
+}
+sourceCompatibility = JavaVersion.VERSION_16
+targetCompatibility = JavaVersion.VERSION_16
```

Comment:
Java 1.6?

Reply:
Not 1.6 - 16.  MC 1.17 [requires Java 1.16](https://minecraft.fandom.com/wiki/Java_Edition_21w19a), and conveniently Gradle 7 does too.